src/javascript/binary/websocket_pages/user/new_account/welcome_page.js

```diff
@@ -16,13 +16,17 @@ const WelcomePage = (() => {
             const show_welcome_msg  = upgrade_info.can_upgrade;
 
             const setButtonLink = (url, msg) => {
-                upgrade_btn.html(createElement('span', { text: localize(msg) })).setAttribute('href', Url.urlFor(url));
+                if(upgrade_btn) {
+                    upgrade_btn.html(createElement('span', { text: localize(msg) })).setAttribute('href', Url.urlFor(url));
+                }
             };
 
             const welcome_msg = document.getElementById('welcome_container');
-            welcome_msg.setVisibility(1);
+            if(welcome_msg) {
+                welcome_msg.setVisibility(1);
+            }
 
-            if (show_welcome_msg) {
+            if (show_welcome_msg && upgrade_btn) {
                 setButtonLink(upgrade_info.upgrade_link, 'Upgrade now');
             } else {
                 upgrade_btn.classList.add('button-disabled');
```

Comment:
here you are saying if `upgrade_btn` doesn't exit, add `button-disabled` class to it, which won't work because the `btn_upgrade` doesn't exist :)

Reply:
Ahh, right, should've put more thought into it. XO